mkdocs.yml

```
@@ -29,8 +29,8 @@ markdown_extensions:
         permalink: true
     - attr_list
     - pymdownx.emoji:
-        emoji_index: !!python/name:materialx.emoji.twemoji
-        emoji_generator: !!python/name:materialx.emoji.to_svg
+        emoji_generator: ""
+        emoji_index: ""
```

I'd keep this change as is. Not sure why they have to be removed.
```suggestion
        emoji_index: !!python/name:materialx.emoji.twemoji
        emoji_generator: !!python/name:materialx.emoji.to_svg
```

---

mhmmm I don't really recall touching this file, making these changes fixed the error.